src/app/public/modules/character-counter/index.ts

```diff
@@ -0,0 +1,2 @@
+export { SkyCharacterCounterIndicatorComponent } from './character-counter-indicator.component';
```

Comment:
I know we've been inconsistent in the past, but we're trying to push for wildcard exports for barrel files.

```
export * from './character-counter-indicator.component';
export * from './character-counter.module';
```